src/components/Article/Article.jsx

```diff
@@ -26,6 +26,10 @@ const Article = () => {
             setArticle(data)
             setIsLoading(false)
         })
+        .catch((error) => {
+            console.log(error)
+            alert('Article does not exist')
```

Comment:
You may not want to do alerts here as it traps the user on the incorrect page. Perhaps render an error message instead of the single article.

Reply:
You could add a back to all articles button like you did for the topics 👍